grafana/dm-apps.json

```diff
@@ -426,7 +426,7 @@
           "targets": [
             {
               "refId": "A",
-              "target": "cloudwatch.application_500s.$environment.$app.500s.samplecount"
+              "target": "cloudwatch.application_500s.$environment.$app.500s.sum"
```

Comment:
So we be putting `:max` on this like the rest of our targets? I notice that it wasn't there before but not sure if that was a mistake on my behalf or intentional. Any idea?

Reply:
Good call, it was defaulting to average without. I've added it on.